charts/perses/templates/tests/test-config.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.testFramework.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-test
+  namespace: {{ .Release.Namespace }}
+  labels:
+    type: helm-chart-test
+  annotations:
+    "helm.sh/hook": test
+    "helm.sh/hook-delete-policy": "before-hook-creation,hook-succeeded"
+data:
+  run.sh: |-
+
+    #!/usr/bin/env bats
+    
+    load "/usr/lib/bats/bats-detik/utils"
+    load "/usr/lib/bats/bats-detik/detik"
+    
+    DETIK_CLIENT_NAME="kubectl"
+    
+    @test "Verify successful deployment and running status of the {{ .Release.Name }}-operator pod" {
```

Comment:
Is the operator suffix right?

Reply:
> Is the operator suffix right?
Nope. Copy paste error 🤗, fixed in 3cc01c3022c398e246f91811e0ea4d030666e83d